Sherlock/yoda/DOCU.md

```diff
@@ -355,11 +369,14 @@ api.Transaction([](PrimesAPI::T_DATA data) {
 // Primes that start with `4`.
 EXPECT_EQ("41,43,47",
           api.Transaction([](PrimesAPI::T_DATA data) {
-            // TODO(dkorolev): In real life, the order can be off. -- D.K.
             std::ostringstream os;
+            std::set<int> n;
             for (const auto cit : data[static_cast<FIRST_DIGIT>(4)]) {
-              os << ',' << (static_cast<int>(cit.row) * 10 +
-                            static_cast<int>(cit.col));
+              n.insert(static_cast<int>(cit.row) * 10 +
+                       static_cast<int>(cit.col));
+            }
+            for (int i : n) {
```

Comment:
Ok, it can accept `std::set<std::string>`, but here I have `std::set<int>` and change will require additional `to_string`.